src/index.js

```diff
@@ -130,7 +130,13 @@ const optionConfig = {
         varName: "NUMBER",
         help: "Give the minimum acceptable I18N score allowed in this run. Valid values are 0-100. Default: no minimum",
         type: validateInt.bind(null, "min-score")
-    }
+    },
+    output: {
+        short: "o",
+        varName: "fileName",
+        "default": "result.txt",
+        help: "Allows you to write the results into a file."
```

Comment:
`.. to write the output into a file."`

Since the output includes more than just the results. It's header + results + summary + footer

Reply:
I updated it